### PR TITLE
fix: MD033/no-inline-html

### PR DESCRIPTION
**Pull request by nschonni**
- Escape inline elements that didn't render
- Ignore elements currently used by Markdown tables
- Encode elements inside pre/code blocks

### CODE REVIEW

**Review by Lxiamail — COMMENTED** [review]
Didn't see any technical concern with WCF related doc updates. Thanks

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks, @nschonni. In the toc.md files, however, the backslash is rendering in the output TOC. And one change was unrelated to the original paragraph.

**Comment by mairaw** [comment]
Unfortunately, the proper Markdown syntax never worked for the TOC files. 

Also, with PR #11443, you'd get a lot of merge conflicts.

**Comment by nschonni** [comment]
I'll fix @rpetrusha's catch, and I'll just rebase to remove all the TOC changes once that other PR lands

File: docs/framework/wcf/troubleshooting-the-getting-started-tutorial.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ ms.assetid: 69a21511-0871-4c41-9a53-93110e84d7fd
 ---
 # Troubleshoot the Get started with Windows Communication Foundation tutorials
 
-This article provides solutions for the most common problems and errors you might face when you follow the steps in the [Tutorial: Get started with Windows Communication Foundation applications](getting-started-tutorial.md). 
+ Visual Studio saves project files in c:\users\\<user name\>\Documents\\<Visual Studio version\>\Projects.
```
[the remark this answers is not in the document]
Comment (nschonni) [reply docs/framework/wcf/troubleshooting-the-getting-started-tutorial.md]: Not sure where that came from, but I undid the change

**Comment by nschonni** [comment]
Actually, just removed the TOC changes since it was simple while reverting the other change

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks, @nschonni. I'll merge your PR now.

**Pull request merged**